### DISCUSSION TITLE
How to get two-channel optical flow images

### DISCUSSION THREAD

**Discussion by BBBBBBob** [microsoft/AirSim#4463] [Support Q&A]
Hi all, 

I have tried to get two-channel optical flow images as an input in the neural network. 
However, I have confronted the same problem in https://github.com/microsoft/AirSim/issues/4414, in which I can only get either a one-channel image or a three-channel image. 
Has anyone obtained the two-channel optical flow image successfully? 
I would be very grateful if you could offer some help.

**Comment by KaiXin-Chen** [comment]
I just had the same issue. The pfm it generates only have one channel. Did you find a solution?
